zeebe/protocol/ignored-changes.json

```diff
@@ -12,6 +12,60 @@
             "matcher": "java",
             "match": "type * { int io.camunda.zeebe.protocol.**::^initialOffset(); }"
           }
+        },
+        {
+          "ignore": true,
+          "code": "java.field.constantValueChanged",
+          "old": "field io.camunda.zeebe.protocol.record.ExecuteCommandRequestDecoder.BLOCK_LENGTH",
```

Comment:
:thought_balloon: We could have a generic "allow changing the block length" change, e.g.

```json
        {
          "justification": "Ignore changes to the constant version when we bump it",
          "code": "java.field.constantValueChanged",
          "new": {
            "matcher": "java",
            "match": "type * { * io.camunda.zeebe.protocol.**::^BLOCK_LENGTH; }"
          }
        }
```

Since increasing it is typically backwards compatible. Reducing it is not backwards compatible however, so maybe having to set an explicit change is a good thing :thinking:

Reply:
it's a good idea indeed. saves some lines and future headache. but as you  mentioned, this way it would be difficult to specify ignores only for higher values